src/test/java/seedu/sellsavvy/logic/commands/ordercommands/DeleteOrderCommandTest.java

```diff
@@ -38,7 +38,6 @@ public void execute_validIndexOrderList_success() {
         Model expectedModel = model.createCopy();
         Person person = model.getFilteredPersonList().get(3);
         model.updateSelectedPerson(person);
-        expectedModel.updateSelectedPerson(person);
```

Comment:
why did you delete this?

Reply:
this line is completely redundant when I was referring to DeleteOrderCommandTest, felt it wasn't exactly worth creating another PR/issue for.